Makefile

```diff
@@ -1,17 +1,20 @@
 # Not file targets.
-.PHONY: help install install-scripts install-conf install-systemd
+.PHONY: help install install-scripts install-conf install-systemd uninstall
 
 ### Macros ###
 SRCS_SCRIPTS	= $(filter-out %cron_mail, $(wildcard usr/local/sbin/*))
-SRCS_CONF	= $(filter-out %template, $(wildcard etc/restic/*))
+SRCS_CONF		= $(filter-out %template, $(wildcard etc/restic/*))
 SRCS_SYSTEMD	= $(wildcard etc/systemd/system/*)
 
 # Just set PREFIX in envionment, like
 # $ PREFIX=/tmp/test make
-DEST_SCRIPTS	= $(PREFIX)/usr/local/sbin
-DEST_CONF	= $(PREFIX)/etc/restic
+DEST_SCRIPT		= $(PREFIX)/usr/local/sbin
```

Comment:
Hi @erikw I've just seen something strange here (I have very little knowledge about make, but) I see you renamed to singular: `DEST_SCRIPT` but, the references below at line 33 and 34 did not get updated accordingly.

I'm guessing this is gonna be a problem, isn't it?

Reply:
Thanks! Fixed in 88f2dc9eca0400f94ff3e84c5de1067dfca694a4